spatialos.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "unity_gdk",
     "project_version": "0.0.1",
-    "sdk_version": "14.6.1"
```

Comment:
This doesn't actually change the worker SDK version used.

To test https://github.com/spatialos/gdk-for-unity/pull/1434, you'll have to update the `gdk.pinned` file to match a commit from that branch (so that you use the GDK with the worker sdk update), and then once the Core PR is merged update the `gdk.pinned` files in FPS/Blank with the commit hash from the `develop` branch.

Reply:
TL;DR - Without updating `gdk.pinned` you're using the GDK as of the 0.3.8 release, and not the version with your worker SDK update.

Reply:
Oh right! I'll update `gdk.pinned` when the core PR is merged 👍